compose.yaml

```diff
@@ -1,35 +1,45 @@
 name: project-name
 
 services:
-###> doctrine/doctrine-bundle ###
+  ###> doctrine/doctrine-bundle ###
   database:
     image: bitnami/mariadb:10.6.19
+    container_name: project-database
     environment:
-      - ALLOW_EMPTY_PASSWORD=yes
-      - MARIADB_USER=app
-      - MARIADB_DATABASE=app
+      - MARIADB_ROOT_PASSWORD=${MARIADB_ROOT_PASSWORD:-root_password}
+      - MARIADB_USER=${MARIADB_USER:-app}
+      - MARIADB_PASSWORD=${MARIADB_PASSWORD:-app_password}
+      - MARIADB_DATABASE=${MARIADB_DATABASE:-app}
     volumes:
-      - ./docker/database:/bitnami/mariadb:rw
+      - database_data:/bitnami/mariadb
```

Comment:
So I can't see my database files anymore in my project directory, even if they are git-ignored?

Reply:
With the updated configuration using the named volume, you will no longer see the database files in your project directory (./docker/database). Named volumes are managed internally by Docker, so the files are stored outside of your project directory and are not directly accessible.

I suggested switching to a named volume because it has several benefits:
- Improved Portability: Docker manages named volumes, making them less prone to host-specific file permission issues.
- Cleaner Project Directory: No database files clutter the project folder, keeping it organized.
- Better Isolation and Management: Named volumes simplify data persistence and backup by keeping data within Docker's control, making it easier to manage across environments.

However, if you need access to the files on the host, a bind mount is still the better choice.

Reply:
It's not a big deal, let's use named volumes.